### DISCUSSION TITLE
Granian 1.0

### DISCUSSION THREAD

**Discussion by mpsantos** [cookiecutter/cookiecutter-django#4812] [Ideas]
Anyone following [this](https://github.com/emmett-framework/granian/releases/tag/v1.0.0)?

It's a rust HTTP server for Python

The benchmarks are impressive: https://github.com/emmett-framework/granian/blob/master/benchmarks/README.md

Looks like a good replacement for Gunicorn and Uvicorn

**Comment by browniebroke** [comment]
First time I'm hearing about it. Very interesting, looks promising!

**Reply by foarsitter** [reply]
Indeed, very interesting! Thanks for sharing @mpsantos  

Using garnian on Heroku in our test environment now without any problems.

**Reply by mpsantos** [reply]
How did you make it run?
I tried changing the command from the start file and had no luck. 
But I was in a hurry and my project runs on a Raspberry, so maybe is that. I didn't try again after that.

**Reply by foarsitter** [reply]
For Heroku I needed to use the `$PORT` variable, but on your RPi you can use a literal value. To access it over the network you need to bind it to `0.0.0.0`.
```
granian --interface wsgi --host 0.0.0.0 --port 8000 config.wsgi
```

**Reply by mpsantos** [reply]
I forgot the binding. It's working now, but you need to add :application at the end:
`granian --interface wsgi --host 0.0.0.0 --port 8000 config.wsgi:application`
